src/styles/base/_typography.scss

```diff
@@ -153,7 +153,7 @@ h1, h2,
 
 h3,
 .h3 {
-	margin-top: 36px;
+	margin-top: 124px;
```

Comment:
it's better if the container has a padding-top instead of having a margin-top in the h3.

Reply:
This issue has been resolved